AcademicReward/PopUps/EditShopItemPage.xaml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<mct:Popup  xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+            xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+            xmlns:mct="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+            x:Class="AcademicReward.EditShopItemPage"
+            CanBeDismissedByTappingOutsideOfPopup="False"
+            Color="White">
+    <!-- Primary Author: Sean Stille -->
+    <!-- Secondary Author: Wil LaLonde -->
+    <!-- Reviewer: Wil LaLonde -->
+    <StackLayout Margin="25">
+        <StackLayout Orientation="Horizontal" Margin="0, 20">
+            <ImageButton
+                Source="blueback.png"
+                WidthRequest="50"
+                HeightRequest="50"
+                Clicked="BackButtonClicked"
+                VerticalOptions="Start"
+                Margin="10"/>
+            <Frame BorderColor="{StaticResource BlueButton}" VerticalOptions="Center" HorizontalOptions="StartAndExpand">
+                <Label
+                    Text="Create Shop Item"
```

Comment:
If this is for editing items, can we change this title to reflect that?

Reply:
Updated text to say "Edit Shop Item